assets/layers/etymology/etymology.json

```diff
@@ -111,26 +111,33 @@
               "stade"
             ],
             "removePostfixes": [
-              "steenweg",
-              "heirbaan",
               "baan",
-              "straat",
-              "street",
-              "weg",
+              "boulevard",
               "dreef",
+              "church",
+              "heirbaan",
+              "gasse",
+              "grundschule",
+              "gymnasium",
+              "kaai",
+              "kerk",
               "laan",
-              "boulevard",
               "pad",
+              "park",
+              "parque",
               "path",
+              "platz",
+              "plaza",
               "plein",
+              "ring",
+              "schule",
               "square",
-              "plaza",
-              "wegel",
-              "kerk",
-              "church",
-              "kaai",
-              "park",
-              "parque"
+              "steenweg",
+              "straat",
+              "straße",
+              "street",
+              "weg",
+              "wegel"
```

Comment:
Could also remove "wegle", I guess. Fairly common in the Swabian-speaking area of Germany.